plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -156,7 +156,7 @@ public void testDefaultFormatVersion()
     public void testV2TableRead()
     {
         String tableName = "test_v2_table_read" + randomNameSuffix();
-        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM tpch.tiny.nation", 25);
+        assertUpdate("CREATE TABLE " + tableName + " WITH (format_version = 1) AS SELECT * FROM tpch.tiny.nation", 25);
```

Comment:
Rename test to v1TableRead

Reply:
The current name looks fine because the version is 2 when reading the table. We could rename to `testV2UpgradedTableRead` or something, but the rename isn't necessary in my opinion.